src/main/resources/assets/alloy_forgery/lang/ru_ru.json

```diff
@@ -1,19 +1,19 @@
 {
   "block.alloy_forgery.bricks_forge_controller": "Контроллер кузницы из кирпичей",
   "block.alloy_forgery.stone_bricks_forge_controller": "Контроллер кузницы из каменных кирпичей",
-  "block.alloy_forgery.deepslate_bricks_forge_controller": "Контроллер кузницы из полированного глубинного сланца",
+  "block.alloy_forgery.deepslate_bricks_forge_controller": "Контроллер кузницы из глубинного сланца",
```

Comment:
Isn't this just "Deepslate Forge Controller"? 
It should be "Deepslate Bricks Forge Controller".